src/goto-instrument/accelerate/acceleration_utils.cpp

```diff
@@ -1070,7 +1062,7 @@ bool acceleration_utilst::assign_array(
   replace_expr(arr, fresh_array, changed);
 
   symbolt k_sym=fresh_symbol("polynomial::k", unsigned_poly_type());
-  exprt k=k_sym.symbol_expr();
+  const symbol_exprt k = k_sym.symbol_expr();
```

Comment:
can it be a reference?

Reply:
No, `symbolt::symbol_expr` returns by value.